client/blocks/post-share/sharing-preview-modal.jsx

```diff
@@ -15,7 +15,7 @@ const SharingPreviewModal = ( props ) => {
 	const {
 		isVisible,
 		onClose,
-		...previewProps,
+		...previewProps
```

Comment:
It shouldn't because it's "spec" behavior that we changed in Babel 7. For context, It was in a v6 patch before and it "broke" react-native so it was moved to a v7 change. And Prettier uses the Babel parser. I think it's mentioned in the description ^

Basically a rest param is always last so it doesn't need a comma

Reply:
TIL, thanks! It is nicely mentioned in the description. I've also verified that prettier handles this correctly.